platform/core/src/services/DisplaySetService/EVENTS.js

```diff
@@ -1,6 +1,7 @@
 const EVENTS = {
   DISPLAY_SETS_ADDED: 'event::displaySetService:displaySetsAdded',
   DISPLAY_SETS_CHANGED: 'event::displaySetService:displaySetsChanged',
+  DISPLAY_SETS_REMOVED: 'event::displaySetService:displaySetsRemoved',
```

Comment:
Do we handle for `ViewportGrid` updating if a displaySet that is currently being viewed is removed? May be worth a follow-up issue that deals with the UI implications here?